code/modules/virus2/effect.dm

```diff
@@ -13,8 +13,8 @@
 		// How damaging the virus is. Higher values are worse.
 
 	var/chance = 3
-		// Under normal conditions, the percentage chance per tick to activate. 
-	var/max_chance = 50	
+		// Under normal conditions, the percentage chance per tick to activate.
+	var/max_chance = 6
```

Comment:
This was already merged into the repo I'm just trying to resolve merge conflicts and it won't work AAAAAAAAAAAAAAAAA